embassy-rp/src/flash.rs

```diff
@@ -887,7 +906,7 @@ pub(crate) unsafe fn in_ram(operation: impl FnOnce()) -> Result<(), Error> {
     }
 
     // Make sure CORE1 is paused during the entire duration of the RAM function
-    crate::multicore::pause_core1();
+    //crate::multicore::pause_core1();
```

Comment:
don't disable this, it'll cause crashes on rp2040.

Reply:
Whoops, this was just a debugging thing I forgot to remove.

Reply:
You should also uncomment crate::multicore::resume_core1() few lines below.